api/tests/unit/application/certification-centers/certification-center-controller_test.js

```diff
@@ -114,4 +117,35 @@ describe('Unit | Controller | certifications-center-controller', () => {
       );
     });
   });
+
+  describe('#findCertificationCenterMembershipsByCertificationCenter', () => {
+
+    const certificationCenterId = 1;
+
+    const request = {
+      params: {
+        certificationCenterId,
+      },
+    };
+
+    beforeEach(() => {
+      sinon.stub(usecases, 'findCertificationCenterMembershipsByCertificationCenter');
+      sinon.stub(certificationCenterMembershipSerializer, 'serialize');
+    });
+
+    it('should call usecase and serializer and return ok', async () => {
```

Comment:
**suggestion**
La description fidèle au contenu du test est
` it('should call serializer and return its value', async () => {`

Si on veut être plus couvrant, je suggère

```javascript

it('should call usecase, pass its content to serializer and return the serialized value  value', async () => {

  const certificationCenterMemberships = [new CertificationCenterMembership()];
  const serializedcertificationCenterMemberships = 'foo';


  usecases.findCertificationCenterMembershipsByCertificationCenter.withArgs({
        certificationCenterId,
      }).resolves(
         certificationCenterMemberships
    );

  certificationCenterMembershipSerializer.serialize.withArgs(
    certificationCenterMemberships
   ).returns(
   serializedcertificationCenterMemberships
);

(...)

expect(response).to.equal(serializedcertificationCenterMemberships);
```